libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1261,7 +1261,7 @@ RC_Channel *RC_Channels::find_channel_for_option(const RC_Channel::aux_func_t op
 // duplicate_options_exist - returns true if any options are duplicated
 bool RC_Channels::duplicate_options_exist()
 {
-    uint8_t auxsw_option_counts[256] = {};
+    uint8_t auxsw_option_counts[308] = {};
```

Comment:
probably worth adding a 'AUX_FUNC_END` enum for this.

Reply:
I agree with this idea. I will change it.

Reply:
I've changed it.